plugins/CoreAdminHome/CustomLogo.php

```diff
@@ -163,6 +163,16 @@ protected static function rewritePath($path)
         return SettingsPiwik::rewriteMiscUserPathWithInstanceId($path);
     }
 
+    public static function hasUserLogo()
+    {
+        return file_exists(Filesystem::getPathToPiwikRoot() . '/' . CustomLogo::getPathUserLogo());
```

Comment:
As you prefer, we can also replace all calls to this method in this file by the constant.
If we add a new method getAbsolutePath, I think it should be in the Filesystem class, isn't it ?